BUILD.gn

```diff
@@ -806,6 +811,39 @@ optional("xml") {
   ]
 }
 
+optional("svg") {
```

Comment:
This looks like a great approach to me - bring in these sources in the `libskia` / `skia.dll` when this option is enabled.